.circleci/config.yml

```diff
@@ -110,7 +110,7 @@ jobs:
             --workdir /home/circleci/project \
             --env JAVA_OPTS='-Xms512m -Xmx2g' \
             --env GRADLE_OPTS='-Xmx3g -Dorg.gradle.daemon=false -Dorg.gradle.jvmargs="-Xmx2g -XX:+HeapDumpOnOutOfMemoryError"' \
-            circleci/android@sha256:71d61d6c21b29948d57120f476a83cc322a280979bce355c5a0ad771293ca380 \
+            circleci/android:api-29-node \
```

Comment:
Ah, much more readable than the hash 😃 

Wonder though, shouldn't we do the same for https://github.com/wordpress-mobile/gutenberg-mobile/blob/a4cdb95b0e4f1553420b36b6826eb3b2a46bc4e8/.circleci/config.yml#L146?

And while at it, can we put that name as a string [pipeline parameter](https://circleci.com/docs/2.0/pipeline-variables/#pipeline-parameters-in-configuration) so it's defined in a single place but used in many?

Reply:
`api-29-node` takes the latest version of the image with that tag, but in this case we need a previous version because the node version is updated in the latest. We don't have to use the same docker image for both because for the one in this PR the node version is not important. But for consistency I will roll back the latest change and update them both to use the same image from a parameter 👍

Reply:
Ah, I see. Note that no strong feelings about having the 2 images be exactly the same. If we end up with two different than the only thing I'd prefer would be to have a pipeline parameter for each to make it clearer which is which and why.